README.md

```diff
@@ -6,6 +6,13 @@ CharSequence formatted = Phrase.from("Hi {first_name}, you are {age} years old."
   .put("first_name", firstName)
   .put("age", age)
   .format();
+
+CharSequence formattedList = ListPhrase.from(
+  "{a} and {b}",
+  "{a}, {b}",
+  "{a}, and {b}")
+  format(1, 2, 3);
+// "1, 2, and 3"
```

Comment:
This block might be worth a bit more explanation, or more examples.  If you start with a simple one (with a single format) and then do one with three, that may suffice.